unime/src/routes/badges/[id]/+page.svelte

```diff
@@ -178,19 +164,19 @@
         </p>
       </div>
 
-      <!-- Metadata (Table: Credential Subject) -->
+      <!-- Contents (Table: Credential Subject) -->
       <div>
-        <p class="pb-2 text-lg font-semibold text-black dark:text-white">{$LL.BADGE.DETAILS.METADATA()}</p>
+        <p class="pb-2 text-lg font-semibold text-black dark:text-white">{$LL.BADGE.DETAILS.CONTENTS()}</p>
         <div
           class="divide-y divide-solid divide-slate-200 rounded-xl border border-slate-200 bg-white dark:divide-slate-600 dark:border-slate-600 dark:bg-dark"
         >
           {#each Object.entries(entries) as entry}
             <div class="flex flex-col items-start px-4 py-[10px]">
               <p class="text-[13px]/[24px] font-medium text-slate-500">{entry[0]}</p>
-              <p class="w-full break-words text-[13px]/[24px] font-medium text-slate-800 dark:text-white">
-                <!-- TODO: this is a hacky way to display nested data, but also to remove enclosing quotes for regular strings -->
-                {JSON.stringify(entry[1]).slice(1, -1)}
-              </p>
+              <div class="w-full break-words text-[13px]/[24px] font-medium text-slate-800 dark:text-white">
+                <!-- TODO: this is a hacky way to display nested data -->
+                <pre class="whitespace-pre-wrap [font-family:inherit]">{JSON.stringify(entry[1], null, 2)}</pre>
```

Comment:
Maybe create a function with one param entry, so we can give the function a name to convey its meaning. Because I'm not sure what this is supposed to do ;)